icu4c/APIChangeReport.md

```diff
@@ -37,55 +43,178 @@ Changed in  ICU 66 (old, new)
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_DIVES_AKURU |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_KHITAN_SMALL_SCRIPT |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_LISU_SUPPLEMENT |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_SYMBOLS_FOR_LEGACY_COMPUTING |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_TANGUT_SUPPLEMENT |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_YEZIDI |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UIndicPositionalCategory::U_INPC_TOP_AND_BOTTOM_AND_LEFT |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_CHORASMIAN |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_DIVES_AKURU |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_KHITAN_SMALL_SCRIPT |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_YEZIDI |  (missing) | StableICU 66| *(Born Stable)* |
 
 ## Added
 
 Added in ICU 66
   
-| File | API | ICU 65 | ICU 66 |
+| File | API | ICU 66 | ICU 66 |
 |---|---|---|---|
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_CHORASMIAN |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_CJK_UNIFIED_IDEOGRAPHS_EXTENSION_G |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_DIVES_AKURU |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_KHITAN_SMALL_SCRIPT |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_LISU_SUPPLEMENT |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_SYMBOLS_FOR_LEGACY_COMPUTING |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_TANGUT_SUPPLEMENT |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_YEZIDI |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UIndicPositionalCategory::U_INPC_TOP_AND_BOTTOM_AND_LEFT |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_CHORASMIAN |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_DIVES_AKURU |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_KHITAN_SMALL_SCRIPT |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_YEZIDI |  (missing) | StableICU 66| *(Born Stable)* |
+| bytestream.h | void icu::ByteSink::AppendU8(const char*, int32_t) |  (missing) | DraftICU 67
```

Comment:
This table does not render after the first row

Reply:
I saw that

Reply:
fixed now?